src/routes/give-away/create.js

```diff
@@ -0,0 +1,51 @@
+import models from '../../models'
+
+export default {
+
+  method: 'post',
+  path: '/giveaways?',
+
+  requireAuthentication: true,
+
+  handler(request, response) {
+
+    // @TODO: This is getting claned up by the orphan job
+    return models.CodexRecordFile.findById('5b1ef57078d795dc2bae3560')
+      .lean()
+      .then((leanMainImage) => {
+        delete leanMainImage._id
+
+        return models.CodexRecordFile.create(leanMainImage)
```

Comment:
Can you do:

```javascript
return new models.CodexRecordFile(leanMainImage).save()
```

instead to be consistent with how I create new records everywhere else? (Constructor vs .create())